Open Judge System/Services/OJS.Services/Data/ParticipantScores/ParticipantScoresDataService.cs

```diff
@@ -12,14 +11,17 @@
     public class ParticipantScoresDataService : IParticipantScoresDataService
     {
         private readonly IEfGenericRepository<ParticipantScore> participantScores;
+        private readonly IEfGenericRepository<Participant> participantRepository;
         private readonly IParticipantsDataService participantsData;
 
         public ParticipantScoresDataService(
             IEfGenericRepository<ParticipantScore> participantScores,
-            IParticipantsDataService participantsData)
+            IParticipantsDataService participantsData,
+            IEfGenericRepository<Participant> participantRepository)
         {
             this.participantScores = participantScores;
             this.participantsData = participantsData;
+            this.participantRepository = participantRepository;
```

Comment:
Can't you use` IParticipantsDataService participantsData` instead of the repository?

Reply:
The  IParticipantsDataService participantsData does not contains methods for Update() and SaveChanges()

Reply:
You can see how other data services make `Update` and add it.

Reply:
Done